pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/Policies.java

```diff
@@ -125,6 +125,8 @@
     @SuppressWarnings("checkstyle:MemberName")
     public String resource_group_name = null;
 
+    @SuppressWarnings("checkstyle:MemberName")
+    public boolean transaction_enable = false;
```

Comment:
This should be "true" by default, otherwise when you upgrade from Pulsar 2.8.0 to Pulsar 2.9.0 you will see Transactions disabled for every namespace.